src/components/BlockedScenarios/BlockedScenarios.tsx

```diff
@@ -73,13 +72,13 @@ const BlockedScenarios = ({ type }: { type: string }) => {
         },
         nonUSD: {
             actionButton: (
-                <Button onClick={openDerivApp} size='lg' textSize={buttonTextSize}>
-                    <Localize i18n_default_text='Create real USD account' />
+                <Button onClick={openLiveChat} size='lg' textSize={buttonTextSize}>
+                    <Localize i18n_default_text='Live chat' />
                 </Button>
             ),
             description: (
                 <Text align='center'>
-                    <Localize i18n_default_text='Please create a Real USD account to access the Deriv P2P marketplace.' />
+                    <Localize i18n_default_text='Please use live chat to contact our Customer Support team for help.' />
```

Comment:
Hi @farrah-deriv , should we change the contentlike this?
"For assistance, please contact our Customer Support team through live chat."

Reply:
`Please use live chat to contact our Customer Support team for help.` is already approved by the content team.